android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -505,60 +515,39 @@ public boolean setKeyboard(String packageID, String keyboardID, String languageI
         KMManager.getLatestKeyboardFileVersion(getContext(), packageID, keyboardID) : null;
     }
 
-    String kbKey = KMString.format("%s_%s", languageID, keyboardID);
-
     setKeyboardRoot(packageID);
```

Comment:
This should address #5418 for `stable-14.0`. 

@keymanapp/testers: Please test per comment: https://github.com/keymanapp/keyman/pull/5407#issuecomment-873907736.